member.php

```diff
@@ -2730,17 +2731,21 @@
 
 		if($mybb->usergroup['caneditprofiles'] == 1)
```

Comment:
You can marge this if with the below one.
```suggestion
		if($mybb->usergroup['caneditprofiles'] == 1 && modcp_can_manage_user($memprofile['uid']))
```